index.js

```diff
@@ -1,18 +1,38 @@
 /***** Deliverable 1 *****/
-const header = document.querySelector("#header")
+const header = document.querySelector("h1#header")
 console.log("Here's your header:", header)
 
 
 /***** Deliverable 2 *****/
 
-
+header.style.color = "green"
 
 /***** Deliverable 3 *****/
-// console.log('This is what the traveler object looks like: ', traveler)
+console.log('This is what the traveler object looks like: ', traveler)
+const travelerName = document.querySelector('h2')
+travelerName.textContent = traveler.name
 
+const travelerNickName = document.querySelector('em')
+travelerName.textContent = traveler.nickname
 
+const travelerPhoto = document.querySelector('img')
+travelerPhoto.src = traveler.photo
 /***** Deliverable 4 *****/
 
+traveler.animalSightings.forEach(function (animalOpject){
+    const outerli = document.createElement('li')
+
+    outerli.innerHTML=`
+    <li data-id="${animalOpject.id}">
+  <p>${animalOpject.description}</p>
+  <img src="${animalOpject.photo}" alt="${animalOpject.photo}"/>
+  <a href="${animalOpject.link}" target="_blank">Here's a video about the ${animalOpject.species} species!</a>
+</li>
+    `
+    const collection = document.querySelector('ul#animals')
+    collection.append(outerli)
 
+})
```

Comment:
great! You don't need the double li though! You can add the data attribute the "artisanal" way.

```
traveler.animalSightings.forEach(obj => {
    const animalSightingsLI = document.createElement('li')
    animalSightingsLI.dataset.id = obj.id

    animalSightingsLI.innerHTML = `
        <p>${obj.description}</p>
        <img src="${obj.photo}" alt="${obj.species}"/>
        <a href="${obj.link}" target="_blank">Here's a video about the ${obj.species} species!</a>
    `

    const animalsUL = document.querySelector('ul#animals')
    animalsUL.append(animalSightingsLI)

})
```